src/pages/island-architecture.md

```diff
@@ -1,5 +1,240 @@
 ---
 layout: ~/layouts/Main.astro
 ---
+<!-- 
+    I have been spending most the day learning more about Island Architecture, wrote plenty of notes, listened to Fred K Schott's interview on Speakeasy(https://www.youtube.com/watch?v=mgkwZqVkrwo) and the interview with Jason Lengstrof (https://www.youtube.com/watch?v=z15YLsLMtu4)
+    Figured I might give writing this a wee go,
 
-# Island Architecture
\ No newline at end of file
+    I wanted to take this from the direction of it being more of a critique of the past and present state of affairs in web dev
+    Post structure:
+    1)Start with an introduction to Islands Arch
+    2)Talk about the different Architectures that can be used in Web-dev
+    3)MVC/StaticSites - SPA's
+    4)Frameworks, get some external links onto the page
+    4)Moving to ESM
+    5)Benefits of ESM
+    6)
+
+ -->
+
+# Island Architecture
```

Comment:
oh wow this is a pretty extensive document! 

This feels a bit more like a great blog post than documentation. I'm a notorious "no fluff" writer when it comes to documentation, so I'm probably -1 on shipping the docs site with such an extensive exploration of architecture (vs. something that is more to the point).

BUT I do really like this, so how about a suggestion: what if we merged this now, but then if we end up not using it in the final docs site we talk about publishing it as a guest blog post instead? It's a fun read for sure.

@aFuzzyBear thoughts?

Reply:
If you're up for this, adding this as an HTML comment explaining the plan would be great so that we don't forget.

Reply:
I love this idea! Maybe we can have a "Deep Dives" section in the docs somewhere. This echoes back to some of the comments in #20 regarding the great styling deep dive by Drew.

Reply:
Im glad you guys liked that write up, 🥰 me and @Hanawa02 were discussing that somewhat and we thought that like fred says keeping that off the main docs as it is just distracting, I would definitelyy be up for a 'Deep Dives' or a Articles Section, this could be something that would allow external contributors to add value to the astro literature,